Tests/test_image_getim.py

```diff
@@ -10,7 +10,14 @@ def test_sanity(self):
         if py3:
             self.assertIn("PyCapsule", type_repr)
 
-        self.assertIsInstance(im.im.id, int)
+
+        try:
+            #py2.6
+            target_types = (int, long)
```

Comment:
How would this fail on non-Py2.6? Isn't it just creating a tuple? Should the `try`/`except` be around the assert?

(Should also include the expected exception for the `except:`. (And a space after `#` in comments.))